### ISSUE TITLE
After upgrade ...   No module named 'pyworxcloud.clouds'

### ISSUE THREAD

**Issue by maury77** [MTrab/landroid_cloud#138]
### Environment:

- Home Assistant version: 2022.6.0
- Landroid_cloud version: 2.0.2
- Operation system: Raspbian

After upgrade I have this error

 Package landroid setup failed. Integration landroid_cloud No module named 'pyworxcloud.clouds'

**Comment by Papi77** [comment]
My integration also broke - getting many errors, eg:

This error originated from a custom integration.

Logger: custom_components.landroid_cloud.config_flow
Source: custom_components/landroid_cloud/config_flow.py:23 
Integration: Landroid Cloud (documentation, issues) 
First occurred: 4:50:05 AM (2 occurrences) 
Last logged: 4:50:05 AM

Unexpected exception
Traceback (most recent call last):
  File "/config/custom_components/landroid_cloud/config_flow.py", line 98, in async_step_import
    await validate_input(self.hass, import_config)
  File "/config/custom_components/landroid_cloud/config_flow.py", line 23, in validate_input
    data[CONF_TYPE] = "worx"
TypeError: 'str' object does not support item assignment

Logger: homeassistant.components.automation.landroid_l_sync_values
Source: helpers/script.py:1718 
Integration: Automation (documentation, issues) 
First occurred: 4:50:49 AM (1 occurrences) 
Last logged: 4:50:49 AM

Sync Values: Error executing script. Invalid data for call_service at pos 3: string value is None for dictionary value @ data['option']

Logger: homeassistant.components.automation.landroid_l_sync_values
Source: components/automation/__init__.py:525 
Integration: Automation (documentation, issues) 
First occurred: 4:50:49 AM (1 occurrences) 
Last logged: 4:50:49 AM

Error while executing automation automation.landroid_l_sync_values: string value is None for dictionary value @ data['option']

**Comment by MTrab** [comment]
> ### Environment:
> * Home Assistant version: 2022.6.0
> * Landroid_cloud version: 2.0.2
> * Operation system: Raspbian
> 
> After upgrade I have this error
> 
> Package landroid setup failed. Integration landroid_cloud No module named 'pyworxcloud.clouds'

What model mower do you have?

**Comment by tontondachu** [comment]
Hello,

Same for me after upgrading to version 2022.6.0. If I install the Landroid_cloud module, it works after the first reboot but if I reboot once again after that, it freezes the HA boot.
By the way, congrats and thank you for the last update! It's a huge one :)

Regards,
Damien

**Comment by MTrab** [comment]
> Hello,
> 
> Same for me after upgrading to version 2022.6.0. If I install the Landroid_cloud module, it works after the first reboot but if I reboot once again after that, it freezes the HA boot. By the way, congrats and thank you for the last update! It's a huge one :)
> 
> Regards, Damien

That does sound like another issue thou.

**Comment by tontondachu** [comment]
> > Hello,
> > Same for me after upgrading to version 2022.6.0. If I install the Landroid_cloud module, it works after the first reboot but if I reboot once again after that, it freezes the HA boot. By the way, congrats and thank you for the last update! It's a huge one :)
> > Regards, Damien
> 
> That does sound like another issue thou.

Oups yes maybe, sorry :(
If that can help:
`2022-06-02 08:24:16 WARNING (SyncWorker_1) [homeassistant.loader] We found a custom integration landroid_cloud which has not been tested by Home Assistant. This component might cause stability problems, be sure to disable it if you experience issues with Home Assistant
2022-06-02 08:25:42 WARNING (MainThread) [homeassistant.bootstrap] Waiting on integrations to complete setup: landroid_cloud
2022-06-02 08:26:42 WARNING (MainThread) [homeassistant.bootstrap] Waiting on integrations to complete setup: landroid_cloud
2022-06-02 08:27:42 WARNING (MainThread) [homeassistant.bootstrap] Waiting on integrations to complete setup: landroid_cloud
2022-06-02 08:28:42 WARNING (MainThread) [homeassistant.bootstrap] Waiting on integrations to complete setup: landroid_cloud
File "/config/custom_components/landroid_cloud/config_flow.py", line 90, in async_step_import
File "/config/custom_components/landroid_cloud/config_flow.py", line 49, in check_for_existing
File "/config/custom_components/landroid_cloud/config_flow.py", line 51, in <genexpr>
2022-06-02 08:29:42 WARNING (MainThread) [homeassistant.bootstrap] Waiting on integrations to complete setup: landroid_cloud
2022-06-02 08:30:42 WARNING (MainThread) [homeassistant.bootstrap] Waiting on integrations to complete setup: landroid_cloud
2022-06-02 08:31:42 WARNING (MainThread) [homeassistant.bootstrap] Waiting on integrations to complete setup: landroid_cloud
2022-06-02 08:32:42 WARNING (MainThread) [homeassistant.bootstrap] Waiting on integrations to complete setup: landroid_cloud
2022-06-02 08:33:42 WARNING (MainThread) [homeassistant.bootstrap] Waiting on integrations to complete setup: landroid_cloud`

**Comment by MTrab** [comment]
@tontondachu  please create a separate issue for this as this is not related

**Comment by MTrab** [comment]
@maury77  I cannot reproduce this error.

Can you provide a debug log and a device model, please?

**Comment by maury77** [comment]
ok the problem is on startup e init module
I set
 custom_components.landroid_cloud: debug
 pyworxcloud: debug

In debug mode trace only line

 [homeassistant.config] Package landroid setup failed. Integration landroid_cloud No module named 'pyworxcloud.clouds' (See /mnt/dietpi_userdata/homeassistant/packages/landroid.yaml:6).

In Landoird yaml is

landroid_cloud:
  email: xxxxx 
  password: xxxx*
  #type: landxcape
  #type: kress

**Comment by MTrab** [comment]
That doesn't look like a valid config!

Correct format is:
```
landroid_cloud:
  - email: xxxxx
    password: xxxx*
```

**Comment by maury77** [comment]
homeassistant@sdomotica:~$ pip3 install pyworxcloud
Looking in indexes: https://pypi.org/simple, https://www.piwheels.org/simple/
Requirement already satisfied: pyworxcloud in ./.pyenv/versions/3.9.10/lib/python3.9/site-packages (1.4.15)

**Comment by maury77** [comment]
I remove pyworxcloud and I reinstall with last versione

pip3 install pyworxcloud
Looking in indexes: https://pypi.org/simple, https://www.piwheels.org/simple/
Collecting pyworxcloud
  Using cached https://www.piwheels.org/simple/pyworxcloud/pyworxcloud-2.0.1-py3-none-any.whl (13 kB)
Collecting paho-mqtt>=1.6.1
  Using cached https://www.piwheels.org/simple/paho-mqtt/paho_mqtt-1.6.1-py3-none-any.whl (75 kB)
Collecting pyOpenSSL>=22.0.0
  Using cached https://www.piwheels.org/simple/pyopenssl/pyOpenSSL-22.0.0-py2.py3-none-any.whl (55 kB)
Requirement already satisfied: cryptography>=35.0 in ./.pyenv/versions/3.9.10/lib/python3.9/site-packages (from pyOpenSSL>=22.0.0->pyworxcloud) (36.0.2)
Requirement already satisfied: cffi>=1.12 in ./.pyenv/versions/3.9.10/lib/python3.9/site-packages (from cryptography>=35.0->pyOpenSSL>=22.0.0->pyworxcloud) (1.15.0)
Requirement already satisfied: pycparser in ./.pyenv/versions/3.9.10/lib/python3.9/site-packages (from cffi>=1.12->cryptography>=35.0->pyOpenSSL>=22.0.0->pyworxcloud) (2.21)
Installing collected packages: paho-mqtt, pyOpenSSL, pyworxcloud
Successfully installed paho-mqtt-1.6.1 pyOpenSSL-22.0.0 pyworxcloud-2.0.1

And the problem is solved 

should the system download the latest version by itself?

**Comment by MTrab** [comment]
Yes it does - and 2.0.1 is not the latest version.
Seems like your system has some issues

**Comment by maury77** [comment]
https://pypi.org/project/pyworxcloud/

2.0.1 in this env
